test/functional/helpers.lua

```diff
@@ -438,4 +439,5 @@ return {
   curbufmeths = curbufmeths,
   curwinmeths = curwinmeths,
   curtabmeths = curtabmeths,
+  ffi = ffi,
```

Comment:
If it's only for the OS check then we can eventually add a helper function like `get_os()`. I never liked the idea of depending on FFI for this--we could just use `nvim` for the purpose instead, e.g. `helpers.eval('has("win32")')`.